deltachat-ios/Controller/Settings/SettingsViewController.swift

```diff
@@ -38,7 +45,7 @@ internal final class SettingsViewController: UITableViewController {
         let cell = UITableViewCell(style: .value1, reuseIdentifier: nil)
         cell.tag = CellTags.chatsAndMedia.rawValue
         cell.textLabel?.text = String.localized("pref_chats_and_media")
-        if #available(iOS 16.0, *) {
+        if #available(iOS 13.0, *) {
```

Comment:
we're using the "minimum required os version fitting all icons"  on purpose. otherwise, eg. on iOS 13 the dialog would end up in having _some_ icons, which looks like a bug. better just show no icons - which is not even noticeable as this [was the default in older Delta Chat versions](https://github.com/deltachat/deltachat-ios/pull/1827) :)

but indeed, the "minimum required os version fitting all icons" would be 16.1

and maybe the comment can be clearer.

Reply:
We could think about adding the icons from SF Symbols to the assets so that everyone can enjoy them?

Until then, icons are shown for iOS 16+ only 👍